app/views/nominations/request_nomination_invite/success.html.erb

```diff
@@ -6,16 +6,10 @@
 
     <h2 class="govuk-heading-m">What happens next</h2>
     <ul class="govuk-list govuk-list--bullet">
-      <li>your school will receive an email with a link to nominate an induction tutor</li>
-      <li>this link will expire in 21 days</li>
+      <li>sent to the email address your school has listed on our register, <%= govuk_link_to "Get information about schools", "https://www.get-information-schools.service.gov.uk/" %></li
```

Comment:
Looks like we dropped a `>` on the closing tag here. I suspect that what Cypress is annoyed about

Reply:
@ethanmills Ah, great spot! I must have copied it over from the old PR incorrect.